students/Craig_Morton/lesson07/html_render.py

```diff
@@ -0,0 +1,130 @@
+# ------------------------------------------------- #
+# Title: Lesson 7, pt 1/2, html render
+# Dev:   Craig Morton
+# Date:  9/14/2018
+# Change Log: CraigM, 9/14/2018, html render
+# ------------------------------------------------- #
+
+
+class Element:
+    """Base Class for HTML tags"""
+    def __init__(self, tag_name='element', content=None, **kwargs):
```

Comment:
per assignment instructions:
```
It should have class attributes for the tag name
```

class attributes are defined for the class like:
```
class Element:
    tag_name = "element"
```
but you have it defined as instance attributes.